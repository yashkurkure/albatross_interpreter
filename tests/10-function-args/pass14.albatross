fun f1 int (a1 int) {
  var a2 int := a1;
  return a2;
}

return f1(473);
