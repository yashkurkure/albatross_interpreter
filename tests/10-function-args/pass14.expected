Function declared "f1" returns int
	Argument "a1" type int position 0
	Local variable "a2" type int position 1
Argument/local read "a1" type int frame position 0
Argument/local read "a2" type int frame position 1
Function called "f1" returns int
